### PR TITLE
two searchbars

### PR DESCRIPTION
**Pull request by oleh-stekh**
[DEMO LINK](https://oleh-stekh.github.io/layout_search-bar-airbnb/)
[TEST REPORT LINK](https://oleh-stekh.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.large-search-icon {
+  height: 19px;
+  width: 19px;
+  align-self: center;
+  margin-left: 25px;
+  margin-top: -1px;
```
Comment (mykhalenych) [inline src/style.css:25]: why it is -1?

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+.large-search-input {
+  align-self: center;
+  border: none;
+  outline: none;
+  color: #3d4e61;
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  line-height: 40px;
+  height: 40px;
+  width: 158px;
```
Comment (mykhalenych) [inline src/style.css:37]: delete it

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  margin-left: 13px;
+  text-indent: 5px;
+  margin-top: -1px;
```
Comment (mykhalenych) [inline src/style.css:40]: the same

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+.small-search-input {
+  align-self: center;
+  border: none;
+  outline: none;
+  color: #3d4e61;
+  font-family: "Avenir", sans-serif;
+  font-size: 14px;
+  line-height: 19px;
+  height: 19px;
+  width: 158px;
```
Comment (mykhalenych) [inline src/style.css:93]: the same

**Review by mykhalenych — CHANGES_REQUESTED** [review]
looks strange
Screenshot 2023-03-14 at 17 27 34

**Review by etojeDenys — CHANGES_REQUESTED** [review]
1. try to follow kebab-case or BEM fo naming classes
2. and cover your img and tag in `label`

**Review by IvanFesenko — APPROVED** [review]

**Pull request closed without merging**